go/types/opcache.go

```diff
@@ -335,10 +335,10 @@ func decodeValue(bs []byte, asValue bool, vr ValueReader) ([]byte, Value) {
 }
 
 // Note that, if 'v' are prolly trees, any in-memory child chunks will be written to vw at this time.
```

Comment:
good catch. i think that probably does need to flush uncommitted child nodes as well.